config/backstop/get-stories.js

```diff
@@ -10,7 +10,7 @@ const STORYNAME_REGEX = /\.add\s*\(\s*['|"].*?['|"],/g;
 
 
 function hasStoryTypes(text) {
-  return text.indexOf('storiesOf') > -1;
+  return text.indexOf('storiesOf') > -1 && text.match(/\/\/.\s?storiesOf/) === null;
```

Comment:
it won't catch `/* ... */` comments, but then again this whole file is questionable :)